wallet-new/src/Cardano/Wallet/API/V1/Migration/Types.hs

```diff
@@ -194,3 +199,9 @@ instance Migrate (Map Core.TxId (V0.CTx, POSIXTime), Word) [V1.Transaction] wher
     eitherMigrate txsMapAndSize = do
         let txsMapValues = elems . fst $ txsMapAndSize
         mapM (eitherMigrate . fst) txsMapValues
+
+instance Migrate V1.TransactionGroupingPolicy V0.InputSelectionPolicy where
```

Comment:
Ditto as above.

Reply:
@sectore it would be nice to add counterpart so that we can write proper roundrip tests (this would test did we actually write v0 -> v1 correctly). 

Without counterpart we are only hoping v0 -> v1 is done correctly

Reply:
After talking to @akegalj and @adinapoli-iohk we've decided to add `Migration` counterparts later.